otterdog/eclipse-thingweb.jsonnet

```diff
@@ -68,6 +68,19 @@ orgs.newOrg('eclipse-thingweb') {
         },
       ],
     },
+    orgs.newRepo('test-things') {
+      allow_update_branch: false,
+      description: "Collection of Things that can be used for testing different IoT protocols, security mechanisms and interaction styles",
+      homepage: "http://thingweb.io",
```

Comment:
Avoid http protocol

Reply:
ty, I was just following the convention from another repo, but I will use https in the future for all homepages and also adapt existing ones.